src/allmydata/test/cli/test_grid_manager.py

```diff
@@ -223,7 +225,7 @@ def test_sign_missing(self):
             )
 
     @skipIf(platform.isWindows(), "We don't know how to set permissions on Windows.")
-    @skipIf(os.getuid() == 0, "cannot test as superuser with all permissions")
```

Comment:
It turns out that this test and all of the others that defined a skip based on `os.getuid` were causing import-time AttributeErrors.  However, when the test suite fails on Windows on GitHub Actions, the CI job is nevertheless marked as passing.  wooo!

This and the similar changes are required to make the test suite green (by making it pass) on CircleCI on Windows.

Reply:
It was me for the NixOS configurations (the upstream Docker images are not well-suited for non-root usage :/ and I didn't want to build a new Docker image that fixes those problems - so we just run the test suite as root in those configs).